src/transformers/models/vit_mae/modeling_tf_vit_mae.py

```diff
@@ -282,17 +314,23 @@ def random_masking(self, sequence: tf.Tensor, noise: tf.Tensor | None = None):
 
         return sequence_unmasked, mask, ids_restore
 
-    def call(self, pixel_values: tf.Tensor, noise: tf.Tensor = None) -> tf.Tensor:
-        embeddings = self.patch_embeddings(pixel_values)
-
+    def call(
+        self, pixel_values: tf.Tensor, noise: tf.Tensor = None, interpolate_pos_encoding: bool = False
+    ) -> tf.Tensor:
+        b, n, h, w = shape_list(pixel_values)
```

Comment:
nit - we should use clear, explicit var names instead of single letters

```suggestion
        batch_size, num_channels, height, width = shape_list(pixel_values)
```